src/website/app/Router.js

```diff
@@ -36,11 +37,14 @@ export default function Router({ demos }: Props) {
       return [...acc, ...pages];
     }, [])
     .map((page, index) => {
+      const title = `${page.title} | Mineral UI`;
```

Comment:
You can just define `canonicalLink` and `pageTitle` in the `pageMeta` object like you do elsewhere. No need for the extra variables if they are not used elsewhere.

```js
const pageMeta = {
  title: `${page.title} | Mineral UI`,
  canonicalLink: `https://mineral-ui.com${page.path}`
};
```